movieApp.js

```diff
@@ -0,0 +1,170 @@
+                option.text = films[i].title;
+                option.value = ' ';
+                // displayFilmInfo(film);
+                dropdown.add(option)
+                // console.log(option)
+                film = films[i]
+                console.log(film)
+                // if(!dropdown.selectedIndex === 0){
+                //     console.log(dropdown.selecetedIndex) 
+                //  }
+            }
+          
+                
+
+
+                defaultOption.onchange = displayFilmInfo(option);
+                // console.log(option)
+
+            // document.addEventListener('input' , function(event){
+            //     if(event.target.selectedIndex !== defaultOption){
+            //         return 
+            //     }
+
+            // })
+
+            // option.addEventListener('change', event=>{
+            //     if(!option[0] === defaultOption){
+            //         console.log()
+            //         option.value = displayFilmInfo(film)
+            //     }
+            // })
+                // if(film === true){
+                //     //  return displayFilmInfo(film);
+                //     // console.log(film);
+                //     displayFilmInfo(film)
+                // }
+            
+              
+        
+        }).catch((error)=>{
+            console.log('Error!')
+        })
+}
+
+// const getFilmInfo = async () =>{
+
+//     await axios
+//     .get(filmsAPI)
+//     .then((response=>{
+//         console.log(response)
+//         let movies = response.data;
+//         console.log(movies)
+//         for(let i = 0; i < movies.length; i++){
+//             // let description = movies[i].description;
+//             // let year = movies[i].release_date
+//             // let title = movies[i].title
+//             if(movies[i])
+//         }
+//     })).catch((err)=>{
+//         console.log('Something is wrong');
+//     })  
+// }
+
+const displayFilmInfo = async (film) =>{
+    //create another div to put all the content of the movie & give it the ID of the movie name
+    const filmInfo = document.createElement('div');
+    filmInfo.id = film.title;
+
+    // creates h3 with movie title
+    const filmTitle = document.createElement('h3');
+    filmTitle.innerText = film.title;
+
+    // shows the date movie was made 
+    const filmYear = document.createElement('p');
+    filmYear.innerText = film.release_date;
+    
+    //creates paragraph with the movie description
+    const filmDescription = document.createElement('p');
+    filmDescription.innerText = film.description;
+    
+    //selecting the empty div and prepping to push new div of info in it
+    const movieInfo = document.querySelector("#movie-info");
+
+    //appending all the things to the DOM
+    filmInfo.appendChild(filmTitle);
+    filmInfo.appendChild(filmYear);
+    filmInfo.appendChild(filmDescription);
+    movieInfo.appendChild(filmInfo);
+
+}
+
+
+
+const userReview = async () =>{
```

Comment:
Not sure why this function is async function since nothing inside this function needs to be async/await.